### PR TITLE
Create Profile Details Page UI 

### LINKED ISSUE #20: FE: Profile Details page
Each Profile should have its own dedicated page that outlines their profile in detail

### LINKED ISSUE #35: IN: Profile details page
Make profile details page dynamic based on user

- feel free to add additional fields to the Profile model to fit the project use case (gallery, cover photo, tagline)

### PR DESCRIPTION
**Pull request by michael-levitski**
### What this PR does (required):
- Creates the user interface for the profile details page
- Closes #20

### Screenshot:
Screenshot 2022-01-24 012202

### Any information needed to test this feature (required):
- visit the `/profile` route

### CODE REVIEW

File: client/src/pages/Profile/ProfilePage.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const ProfilePage = () => {
+  // const { loggedInUser } = useAuth();
+  // const history = useHistory();
+
+  // if (loggedInUser === undefined) return <CircularProgress />;
+  // if (!loggedInUser) {
+  //   history.push('/login');
+  //   return <CircularProgress />;
+  // }
```
Comment (sundayezeilo) [inline client/src/pages/Profile/ProfilePage.tsx]: Please let's get rid of unused codes

**Review by sundayezeilo — COMMENTED** [review]
Looks good!

File: client/src/components/ProfileCard/Gallery.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { ImageListItem, ImageList } from '@mui/material';
+
+interface PropTypes {
+  images: string[];
+}
+
+const Gallery = ({ images }: PropTypes) => {
+  return (
+    <ImageList cols={4} gap={10}>
```
Comment (rajivtitus) [inline client/src/components/ProfileCard/Gallery.tsx:9]: 👍

File: client/src/components/RequestForm/RequestForm.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <Grid
+          container
+          sx={{ minHeight: '300px' }}
+          direction="column"
+          alignItems="center"
+          justifyContent="space-evenly"
+        >
+          <Typography variant="h6">${profile.hourlyRate}/hr</Typography>
+          <Grid item>
+            <StarRating rating={profile.averageRating} />
```
Comment (rajivtitus) [inline client/src/components/RequestForm/RequestForm.tsx]: Good thing but MUI has a component for ratings, we can use that here instead

File: client/src/pages/Profile/ProfilePage.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import { CircularProgress, Grid } from '@mui/material';
+import { useHistory } from 'react-router-dom';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import ProfileCard from '../../components/ProfileCard/ProfileCard';
+import RequestForm from '../../components/RequestForm/RequestForm';
+import { useAuth } from '../../context/useAuthContext';
+import { Profile } from '../../interface/Profile';
+
+const profile: Profile = {
```
Comment (rajivtitus) [inline client/src/pages/Profile/ProfilePage.tsx:9]: 👍

**Review by rajivtitus — COMMENTED** [review]
Excellent with Michael! Just a small comment to address

**Comment by rajivtitus** [comment]
@michael-levitski LGTM 👍🏽

**Review by rajivtitus — APPROVED** [review]

**Pull request merged**